### PR TITLE
Adding admin-registered users

### LINKED ISSUE #52: Admin-registered users
Invite users (possibly without personal and/or educational data).

There are packages to require the first passwords from emails...

### PR DESCRIPTION
**Pull request by viktorcsimma**
Addresses #52.

### CODE REVIEW

**Comment by viktorcsimma** [comment]
Setting roles is not yet possible; I will add that soon.
Also, could you check whether e-mail sending is functional?

File: resources/lang/hu/registration.php [hunk 1]
```diff
@@ -6,10 +6,15 @@
...
+    'invite_button' => 'Létrehozás és meghívó küldése',
+    'invite_instructions' => 'Itt lehet adott e-mail-címmel egy felhasználót készíteni, és meghívót küldeni a címre.',
     'name' => 'Név',
     'new_password' => 'Új jelszó',
     'old_password' => 'Régi jelszó',
     'password' => 'Jelszó',
     'remember' => 'Emlékezz rám',
+    'successfully_invited' => 'Elküldtünk egy meghívót a megadott e-mail-címre.',
     'tenant_to_collegist' => 'Ez az oldal a Collegium vendégeinek szól. Ha Collegium tagja vagy (bentlakó vagy bejáró), akkor kattints ide!',
+    'used_email' => 'A megadott e-mail-címmel már létezik felhasználó.',
```
Comment (kdmnk) [inline resources/lang/hu/registration.php]: unnecessary (validator does this)

File: app/Http/Controllers/Secretariat/RegistrationsController.php [hunk 2]
```diff
@@ -92,6 +92,21 @@ public function reject(Request $request)
         return redirect()->route('secretariat.registrations')->with('message', __('general.successful_modification'));
     }
 
+    public function invite(Request $request)
+    {
+        $user=new User();
+        $user->name=$request->name;
+        $user->email=$request->email;
+        $user->password=\Hash::make(\Str::random(32));
```
Comment (kdmnk) [inline app/Http/Controllers/Secretariat/RegistrationsController.php]: $user = User::create(['name' => ..., ...])

File: app/Http/Controllers/Secretariat/RegistrationsController.php [hunk 3]
```diff
@@ -92,6 +92,21 @@ public function reject(Request $request)
         return redirect()->route('secretariat.registrations')->with('message', __('general.successful_modification'));
     }
 
+    public function invite(Request $request)
+    {
```
Comment (kdmnk) [inline app/Http/Controllers/Secretariat/RegistrationsController.php:85]: $request->validate(RegisterController::USER_RULES*)

*without password

File: resources/views/secretariat/user/list_users_component.blade.php [hunk 4]
```diff
@@ -56,6 +56,35 @@
...
+            <form method="POST" action="{{ route('secretariat.registrations.invite') }}">
+                @csrf
+                <div class="row">
+                    <div class="col s12">
+                        <blockquote>@lang('registration.invite_instructions')</blockquote>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col s12 m12 l4">
+                        <x-input.text  id="name" type="text" required :value="config('custom.kkt')" />
```
Comment (kdmnk) [inline resources/views/secretariat/user/list_users_component.blade.php]: remove :value fields

File: app/Http/Controllers/Secretariat/RegistrationsController.php [hunk 5]
```diff
@@ -92,6 +92,21 @@ public function reject(Request $request)
...
+        $user=new User();
+        $user->name=$request->name;
+        $user->email=$request->email;
+        $user->password=\Hash::make(\Str::random(32));
+        try {
+            $user->save();
+            \Invytr::invite($user);
+            return redirect()->back()->with('message', __('registration.successfully_invited'));
```
Comment (kdmnk) [inline app/Http/Controllers/Secretariat/RegistrationsController.php]: Just redirect to `secretariat.permissions.show`, ['id' => $user->id]

File: app/Http/Controllers/Secretariat/RegistrationsController.php (hunk 2 above)
Comment (kdmnk) [reply app/Http/Controllers/Secretariat/RegistrationsController.php]: and verified => true

**Review by kdmnk — CHANGES_REQUESTED** [review]

**Comment by viktorcsimma** [comment]
I now check the username and the address; is that OK?

**Comment by kdmnk** [comment]
I have added a custom mail view for this.
Thanks, the emails are sent, I will release this

**Review by kdmnk — APPROVED** [review]

**Pull request merged**